src/data/getRepos.tsx

```diff
@@ -0,0 +1,35 @@
+import memoizeOne from 'memoize-one'
+
+import client from '../apollo-client'
+import { useFragment as asFragment } from '../gql/fragment-masking'
+
+import { REPOS_QUERY, RepoFragment } from './queries/recipesQueries'
+
+import type { RepoFragmentFragment, ReposQuery } from '../gql/graphql'
+
+export const reposCache: {
+  filtered: RepoFragmentFragment[]
+} = {
+  filtered: [],
+}
+const filterRepos = memoizeOne((data: ReposQuery) => data?.repositories?.edges
+  ?.map(edge => edge?.node)
+  .map(node => asFragment(RepoFragment, node))
+  .filter((repo: RepoFragmentFragment | undefined | null): repo is RepoFragmentFragment => !!repo && !repo.private))
```

Comment:
I don't believe this query will ever return private repos, were you seeing that when unauthenticated?

Reply:
Ah, I did almost all dev work while authenticated and and never changed this.